src/util-hash-lookup3.c

```diff
@@ -64,8 +64,6 @@ on 1 byte), but shoehorning those bytes into integers efficiently is messy.
 # define HASH_BIG_ENDIAN 0
 #endif
 
-#define hashsize(n) ((uint32_t)1<<(n))
```

Comment:
lets leave this file as close to vanilla as possible

Reply:
What do you mean by "close to vanilla" ?

Reply:
this is a file we vendored in, so lets keep it as close to the original as possible

Reply:
But this triggers rightfully the warning.

Is there some upstream to fix ?